pkg/project/project.go

```diff
@@ -43,18 +42,16 @@ func Create(client *occlient.Client, projectName string, wait bool) error {
 }
 
 // Delete deletes the project with name projectName and returns errors if any
-func Delete(client *occlient.Client, projectName string) error {
-	// Loading spinner
-	s := log.Spinnerf("Deleting project %s", projectName)
-	defer s.End(false)
+func Delete(client *occlient.Client, projectName string, wait bool) error {
+	if projectName == "" {
```

Comment:
I think the cobra package handles this condition for us already. Is this check going to be helpful to IDE plugins folks? If not, we could remove it, I guess.
```sh
$ odo project delete --wait
Error: accepts 1 arg(s), received 0
Usage:
  odo project delete [flags]
```

Reply:
Actually, this is for nil pointer checking / when using the test suite. 

Yes, Cobra covers it. But this was added in case the function was used somewhere else / testing. So we don't encounter any nil point errors.